build_tools/github_actions/amdgpu_family_matrix.py

```diff
@@ -26,7 +26,7 @@
             "bypass_tests_for_releases": True,
         },
     },
-    "gfx115x": {
+    "gfx1151": {
```

Comment:
We need to decide is gfx1151 and gfx1150 will be grouped or separate, and there are many other places to update to add the target. See https://github.com/ROCm/TheRock/issues/1337, where there was a request to release gfx115X-igpu.